test/unit/int_algolia/scripts/algolia/steps/algoliaProductDeltaIndex.test.js

```diff
@@ -90,15 +109,17 @@ describe('process', () => {
         expect(algoliaOperations).toMatchSnapshot();
     });
 
-    test('master-level indexing', () => {
-        global.customPreferences['Algolia_RecordModel'] = 'master-level';
```

Comment:
Why removing the existing test? It's important to test the most cases if possible.